addons/mail/static/src/core/common/message_model.js

```diff
@@ -62,6 +62,18 @@ export class Message extends Record {
     /** @type {string} */
     scheduledDatetime;
     starredPersonas = Record.many("Persona");
+    onlyContainsEmojis = Record.attr(false, {
+        compute() {
+            const div = document.createElement("div");
+            div.innerHTML = this.body;
+            const bodyWithoutTags = div.textContent;
+            const withoutEmojis = bodyWithoutTags.replace(
+                /\p{Emoji_Presentation}|\p{Emoji}\uFE0F|\u200d/gu,
```

Comment:
There are multiple instances of this regexp in the code:
* https://github.com/odoo/odoo/blob/master/addons/mail/static/src/core/common/thread_service.js#L615
* https://github.com/odoo/odoo/blob/master/addons/mail/static/src/js/emojis_mixin.js#L20

Maybe it should be an exported `const`?